src/app/pages/send/ordinal-inscription/hooks/use-send-inscription-form.tsx

```diff
@@ -100,17 +103,20 @@ export function useSendInscriptionForm() {
 
       const { hex } = resp;
       const feeRowValue = formFeeRowValue(values.feeRate, isCustomFee);
-      return navigate(RouteUrls.SendOrdinalInscriptionReview, {
-        state: {
-          fee: feeValue,
-          inscription,
-          utxo,
-          recipient: values.recipient,
-          time,
-          feeRowValue,
-          tx: hex,
-        },
-      });
+      return navigate(
```

Comment:
I need to look into this more as if I don't compose the route like this I get a weird error when trying to send

![Screenshot 2023-10-27 at 16 24 47](https://github.com/leather-wallet/extension/assets/2938440/d35e6cdc-2fc7-49fa-8f69-5a5408104f08)